frontend/src/router/routeHandlers.ts

```diff
@@ -148,5 +148,6 @@ export const beforeEachRouteHandler = async (
     if (AuthService.state.value.famLoginUser) {
         // condition needed to prevent infinite redirect
         await AuthService.methods.refreshToken();
+        await AuthService.methods.getUserAccess();
```

Comment:
Add here, so in the case if a user already logged in,  and he has been granted a new admin role, then when refresh the token, and call get access again, the user should be able to manage the new app

Reply:
Thanks Catherine for this addition!
It is being called twice after login, but I think it is ok. When deployed, we can see if there is impact.
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/f9ffac8d-fe1b-49e8-a81b-226c0a8addea)

Reply:
yeah I noticed that as well, it's been called when handle post login, that's when we call both refresh toke and get user access, and when enter each router, it will call refresh toke and get user access again

Reply:
yeah let's see if there is any impact in dev, thanks so much!!